src/components/ScrollView/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+import ScrollView, { ScrollViewProps } from '..'
+
+export default {
+  component: ScrollView,
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'A component that will add a scroll bar on the left when text overpass box height.',
+      },
+    },
+  },
+  title: 'Components/ScrollView',
+} as Meta
+
+const Template: Story<ScrollViewProps> = args => (
+  <ScrollView height="300px" {...args}>
```

Comment:
As ScrollView does not accept xstyled props anymore it seems that this example is not working

Maybe ?
```suggestion
  <ScrollView style={{ height: "300px" }} {...args}>
```